$NetBSD: patch-ac,v 1.1 2003/11/19 11:54:48 agc Exp $

Allow the configure script to find the buildlink'ed freetype-config script

--- configure	2003/11/19 11:35:05	1.1
+++ configure	2003/11/19 11:36:42
@@ -1935,7 +1935,7 @@
   ;;
   *)
   as_save_IFS=$IFS; IFS=$PATH_SEPARATOR
-for as_dir in ${prefix}/bin ${exec_prefix}/bin /usr/local/bin /opt/local/bin /usr/bin /usr/X11/bin
+for as_dir in $PATH
 do
   IFS=$as_save_IFS
   test -z "$as_dir" && as_dir=.
